tests/ProjectsTest.php

```diff
@@ -1738,7 +1740,7 @@ public function testMembershipRoleChange()
 
     public function testPayAsYoGoDetails()
     {
-        $feature = 'pay-as-you-go';
+        $feature = self::PAY_AS_YOU_GO_CREDITS_ADMIN_FEATURE_NAME;
```

Comment:
Tady je asi špatně const
```suggestion
        $feature = self::PAY_AS_YOU_GO_PROJECT_FEATURE_NAME;
```

Reply:
fixed